Clinical-Genomics/scout
# Panel maintainers - fix #705

Gene panels can now have panel maintainers
Newly created genes are initially maintained by their creator. Admins can always maintain.

**How to test**:
1. modify, add, remove, create new, replace genes and what not as maintainer and non-maintainer. 
2. Old panels should work as before until maintainers are manually added to the db. 

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
